dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -549,7 +550,7 @@ def __init__(self, exec_func, **kwargs):
         import numpy as np
         seed = kwargs.get('seed', -1)
         if seed < 0:
-            seed = np.random.randint(0, 2**32)
+            seed = np.random.randint(sys.maxsize)
```

Comment:
Hmm, wouldn't this be platform dependent? Maybe used iinfo and fixed sized type?

Reply:
I set a fixed value at `(1 << 31) - 1`